pkg/parser/listener.go

```diff
@@ -1,10 +1,10 @@
-package nlurules
+package parser
```

Comment:
@nori266 I don't think that Listener is part of the parsing logic. Rather, it's part of the grammar handling logic, so it should reside in `internal/grammar`.